frameworks/cassandra/tests/test_recovery.py

```diff
@@ -27,6 +27,17 @@ def configure_package(configure_universe):
 
 @pytest.mark.sanity
 @sdk_utils.dcos_1_9_or_higher # dcos task exec not supported < 1.9
+def test_node_replace_replaces_seed_node():
```

Comment:
Both of the proposed changes seem to introduce more resiliency for good reasons.